### PR TITLE
Add: martinchrzan.ColorPicker version 1.0.9.0

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/22471)

### CODE REVIEW

File: manifests/m/martinchrzan/ColorPicker/1.0.9.0/martinchrzan.ColorPicker.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Created using YamlCreate.ps1 v1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: martinchrzan.ColorPicker
+PackageVersion: 1.0.9.0
```
Comment (ImJoakim) [inline manifests/m/martinchrzan/ColorPicker/1.0.9.0/martinchrzan.ColorPicker.installer.yaml:5]: image

```suggestion
PackageVersion: 1.0.9
```

File: manifests/m/martinchrzan/ColorPicker/1.0.9.0/martinchrzan.ColorPicker.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Created using YamlCreate.ps1 v1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: martinchrzan.ColorPicker
+PackageVersion: 1.0.9.0
```
Comment (ImJoakim) [inline manifests/m/martinchrzan/ColorPicker/1.0.9.0/martinchrzan.ColorPicker.locale.en-US.yaml:5]: image

```suggestion
PackageVersion: 1.0.9
```

File: manifests/m/martinchrzan/ColorPicker/1.0.9.0/martinchrzan.ColorPicker.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using YamlCreate.ps1 v1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: martinchrzan.ColorPicker
+PackageVersion: 1.0.9.0
```
Comment (ImJoakim) [inline manifests/m/martinchrzan/ColorPicker/1.0.9.0/martinchrzan.ColorPicker.yaml:5]: image

```suggestion
PackageVersion: 1.0.9
```

**Comment by ImJoakim** [comment]
Remember to update the version folder as well 👍

**Comment by ImJoakim** [comment]
> Install MSI once and then check. 👍

Ah, I see. That's an odd way of doing it.

**Review by OfficialEsco — APPROVED** [review]
https://github.com/microsoft/winget-pkgs/pull/22408#issuecomment-886911437
@ImJoakim @ItzLevvie @jedieaston
This is a temporary workaround until Manifest Schema 1.1.0, winget-cli actually don't look at .0's as a new version so this does not cause a issue
```
PS C:\Users\WDAGUtilityAccount\Desktop\winget-pkgs> winget list
Name                                    Id                                           Version          Available   Source
------------------------------------------------------------------------------------------------------------------------
ColorPicker                             martinchrzan.ColorPicker                     1.1.2
Microsoft Edge                          Microsoft.Edge                               89.0.774.68      92.0.902.55 winget
Microsoft Edge Update                   Microsoft Edge Update                        1.3.141.63
App Installer                           Microsoft.DesktopAppInstaller_8wekyb3d8bbwe  1.12.11692.0
Microsoft Edge                          Microsoft.MicrosoftEdge.Stable_8wekyb3d8bbwe 89.0.774.68
Windows Package Manager Source (winget) Microsoft.Winget.Source_8wekyb3d8bbwe        2021.727.911.750
PS C:\Users\WDAGUtilityAccount\Desktop\winget-pkgs> winget upgrade
Name           Id             Version     Available   Source
------------------------------------------------------------
Microsoft Edge Microsoft.Edge 89.0.774.68 92.0.902.55 winget
```
This is somewhat the same issue Microsoft Teams and GitHub Desktop has

**Pull request merged**